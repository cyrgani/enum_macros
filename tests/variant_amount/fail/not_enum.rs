use enum_macros::VariantAmount;

#[derive(VariantAmount)]
struct S {
    a: u32,
    b: u8,
}

#[derive(VariantAmount)]
union U {
    a: u32,
    b: f32,
}

fn main() {}