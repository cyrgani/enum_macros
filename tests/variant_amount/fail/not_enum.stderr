error: expected `enum`
 --> tests/variant_amount/fail/not_enum.rs:4:1
  |
4 | struct S {
  | ^^^^^^

error: expected `enum`
  --> tests/variant_amount/fail/not_enum.rs:10:1
   |
10 | union U {
   | ^^^^^
